lib/api.rb

```diff
@@ -157,19 +218,31 @@ def validation_error(msg)
         return validation_error("Unknown error")
       end
 
+      headers "access-control-allow-origin" => "*"
+
       {
         :access_token => d.access_token,
         :expires_in => (d.token_expires_at - Time.now).floor,
         :token_type => "Bearer",
         :refresh_token => d.refresh_token,
         :Key => d.user.key,
+        :PrivateKey => d.user.private_key,
         # TODO: when to include :privateKey and :TwoFactorToken?
       }.to_json
```

Comment:
add the PrivateKey to user.to_hash and then just use this instead of writing out the hash manually.